config/routes.rb

```diff
@@ -21,6 +21,8 @@
 
     get '/*route/folder_new', to: 'folders#new', as: :new_folder
     get '/*route/:binary_name', to: 'folders/binaries#show', as: :binary, format: true
+    post '/*route/:binary_name/like', to: 'folders/binaries/likes#create', as: :like_file, format: true
```

Comment:
@case-eee this is one way i tried to do routes, which got me closest to what i am thinking